_includes/config_options.html

```diff
@@ -0,0 +1,27 @@
+{% assign tableFileParam = {{include.file}} %}
+{% assign tableFile = site.data[tableFileParam] %}
+
+<table class="table">
+  <caption>Configuration Options</caption>
+  <thead>
+    <tr>
+      <th scope="col">Config Path</th>
+      <th scope="col">Possible Value(s)</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+  {% for entry in tableFile %}
+    {% assign current_name = entry.name %}
```

Comment:
Nice. Can the _Code Path_ lines be in style `<code></code>` automatically?

Reply:
I don't understand the question.

Reply:
Are you asking about the usage of backticks? As it is HTML it does not translate a backtick into `<code>` so code needs to be used explicitly.